runtime/lua/vim/lsp/util.lua

```diff
@@ -1699,7 +1494,7 @@ function M.open_floating_preview(contents, syntax, opts)
   local floating_winnr = api.nvim_open_win(floating_bufnr, false, float_option)
   if do_stylize then
     vim.wo[floating_winnr].conceallevel = 2
-    vim.wo[floating_winnr].concealcursor = 'n'
+    vim.bo[floating_bufnr].filetype = 'markdown'
```

Comment:
Does this also need a `vim.treesitter.start(...)` call in case users haven't set it up?

Reply:
It does ;)

Reply:
Yes, it does ;)